taskcluster/ci/split-mono-src/kind.yml

```diff
@@ -66,17 +62,12 @@ task-defaults:
             - -c
               # TODO: move number of chunks elsewhere
             - >-
-                export BIN=$MOZ_FETCHES_DIR &&
-                $VCS_PATH/taskcluster/scripts/pipeline/split-mono-taskcluster.sh
-                {split_chunks}
-                /builds/worker/artifacts
-                {split_length}
+                python3 $VCS_PATH/pipeline/translate/splitter.py
+                --output_dir=/builds/worker/artifacts
+                --num_parts={split_chunks}
+                --compression_cmd=zstdmt
```

Comment:
Praise: This feels much simpler and easier to maintain. Thanks for taking the time to do this. I'm happy to get rid of some of the environment variables as well.